include/Lv2UridMap.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Lv2UridMap.cpp - Lv2UridMap class
+ *
+ * Copyright (c) 2019 Johannes Lorenz <j.git$$$lorenz-ho.me, $$$=@>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#ifndef LV2URIDMAP_H
+#define LV2URIDMAP_H
+
+#include "lmmsconfig.h"
+
+#ifdef LMMS_HAVE_LV2
+
+#include <lv2/urid/urid.h>
```

Comment:
On Ubuntu 16.04, the path is `lv2/lv2plug.in/ns/ext/urid/urid.h`. We should somehow resolve the path by some CMake logic.

Reply:
Thanks for the hint. I just saw a CI error caused by this.

On Manjaro, `lv2/lv2plug.in/ns/ext/urid/urid.h` is also one valid path (it's a symlink). Since jalv uses this path, too, I changed it now. Let's wait for CI.

Reply:
Cool, that fixed the CI error.